engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -65,15 +98,38 @@ public void resetCamera() {
     }
 
```

Comment:
@emanuele3d: As far as the command/keyboard question goes the `@Command` annotation itself registers the command as available in the game console by its method name, no additional action needed. Binding it to a hotkey would have to be done separately, which is another annotation to define the hotkey then simply have its execution run the command's method :-)